### PR TITLE
Add Sapphire ParaTime node tab

### PR DESCRIPTION
**Pull request by aefhm**
## Description

Add note for running Sapphire ParaTime.

### CODE REVIEW

File: docs/get-involved/run-node/paratime-node.mdx [hunk 1]
```diff
@@ -112,6 +112,40 @@ committee on Mainnet, your entity needs to:
...
+A confidential EVM-compatible Oasis Foundation developed ParaTime that enables
+the use of EVM smart contracts on the Oasis network.
+
+### Overview
+
+* **Leading Developer:** [Oasis Protocol Foundation], with contributions from
+  community developers
+* **Status:** Deployed on Mainnet and Testnet
+* **Testnet Launch Date:** July 2022
+* **Mainnet Launch Date:** May 2023
```
Comment (matevz) [inline docs/get-involved/run-node/paratime-node.mdx]:
```suggestion
* **Mainnet Launch Date:** December 2022
```

File: docs/get-involved/run-node/paratime-node.mdx [hunk 2]
```diff
@@ -112,6 +112,40 @@ committee on Mainnet, your entity needs to:
 - Have a validator in the **validator set**.
 - Have at least **5,000,000.00 ROSE staked/delegated** to it.
 
+</TabItem>
+<TabItem value="Sapphire ParaTime">
```
Comment (kostko) [inline docs/get-involved/run-node/paratime-node.mdx]: Should we make Sapphire the first tab?

File: docs/get-involved/run-node/paratime-node.mdx [hunk 3]
```diff
@@ -112,6 +112,40 @@ committee on Mainnet, your entity needs to:
...
+* **Requires SGX:** yes
+* **Parameters:**
+  * [Mainnet](../../node/mainnet/README.md#sapphire-paratime)
+  * [Testnet](../../node/testnet/README.md#sapphire-paratime)
+
+### Features
+
+* Fully decentralized with node operators distributed across the world.
+* Oasis **ROSE** tokens are the native token used in the ParaTime for gas fees.
+
```
Comment (matevz) [inline docs/get-involved/run-node/paratime-node.mdx:131]:
```suggestion
* Support for confidential compute.

```

File: docs/get-involved/run-node/paratime-node.mdx (hunk 1 above)
Comment (kostko) [inline docs/get-involved/run-node/paratime-node.mdx]:
```suggestion
* **Mainnet Launch Date:** Dec 2022
```

File: docs/get-involved/run-node/paratime-node.mdx (hunk 2 above)
Comment (matevz) [inline docs/get-involved/run-node/paratime-node.mdx]: Let's change the order of tabs so it's consistent with https://docs.oasis.io/dapp/:
1. Sapphire
2. Emerald
3. Cipher

**Review by matevz — APPROVED** [review]
Nice catch. Some minor nits, otherwise fine!

**Comment by aefhm** [comment]
> Nice catch. Some minor nits, otherwise fine!

'twas @gw0

File: docs/get-involved/run-node/paratime-node.mdx (hunk 2 above)
Comment (aefhm) [reply docs/get-involved/run-node/paratime-node.mdx]: It's like you and @matevz are on the same page. ❤️

**Pull request merged**